test/gateways/monei_test.exs

```diff
@@ -196,6 +197,12 @@ defmodule Gringotts.Gateways.MoneiTest do
     then = Enum.map(all, &Gateway.respond({:ok, &1}))
     assert Keyword.keys(then) == [:ok, :error, :error, :error]
   end
+
+  defp amount(amount, currency \\ "USD") do
+    %{amount: Decimal.new(amount), currency: currency}
```

Comment:
@oyeb instead of using a simple map, use `ex_money` struct example here, put `ex_money` as only a dependency for test environment. 

Or you also write 3 test cases for testing 3 existing libs by passing `money` structs of these libs showing how we accept all the libraries in elixir.

cc @kartikjagdale @arjun289 @SagarKarwande